tests/functional/test_home.py

```diff
@@ -7,16 +7,13 @@
 from pages.home import HomePage
 
 
-@pytest.mark.skip_if_internet_explorer(reason='https://ci.us-west.moz.works/job/bedrock_integration_tests_runner/11169/')
-@pytest.mark.skip_if_firefox(reason='Download button is not shown to Firefox browsers.')
```

Comment:
Also removing this skip, since we no longer hide the home page download button for Firefox users 😸